src/middlewares/userValidation.js

```diff
@@ -0,0 +1,38 @@
+import joiValidator from './joiValidator';
+import Joi from '@hapi/joi';
+import Util from '../utils/response';
+const util = new Util();
+
+/**
+ * Helper function to validate user profile
+* @param {Object} req - server request
+* @param {Object} res - server response
+* @returns {Object} - custom response
+* @param {Function} next
+*/
+export const validateProfileData = async (req, res, next) => {
+  try {
+    const schema = Joi.object().keys({
+      firstName: Joi.string().min(2),
+      lastName: Joi.string().min(2),
+      gender: Joi.string(),
```

Comment:
Gender should only be `male` and `female`.
Phone numbers should be 11 numbers.

Reply:
okay,
It has been fixed

Reply:
Okay, It has been fixed